bitwarden_license/bit-web/src/app/secrets-manager/layout/layout.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="tw-flex tw-w-full">
   <nav
-    class="tw-fixed tw-max-h-screen tw-min-h-screen tw-w-60 tw-overflow-auto tw-bg-background-alt3"
+    class="tw-fixed tw-inset-y-0 tw-left-0 tw-max-h-screen tw-min-h-screen tw-w-60 tw-overflow-auto tw-bg-background-alt3"
```

Comment:
```suggestion
    class="tw-fixed tw-inset-y-0 tw-left-0 tw-h-screen tw-w-60 tw-overflow-auto tw-bg-background-alt3"
```
Nitpick - You could technically just set `tw-h-screen` and achieve the same result as adding `tw-max-h-screen tw-min-h-screen`